### ISSUE TITLE
Accessibility swarm on ukrainian/ukraine_in_russian IDX page

### ISSUE THREAD

**Issue by DenisHdz** [bbc/simorgh#7084]
**Is your feature request related to a problem? Please describe.**
Carry out a two devs accessibility swarm for the IDX page 
https://www.test.bbc.com/ukrainian/ukraine_in_russian?renderer_env=live

**Describe the solution you'd like**
The main things to check are: 
- Language changes between `Script Link`, `Header`, `Footer` and the rest of the page
- Accent between Ukrainian and Russian

ACC:
```
Given I'm on the url `ukrainian/ukraine_in_russian`
when I see the page
Then I see the title of the page as a H1 visually displayed

Given I'm on the url `ukrainian/ukraine_in_russian`
when I see the page
Then it is rendered following the same rules as the homepage (except the h1 is visually displayed)
And the content is rendered from https://ares-api.api.bbci.co.uk/api/index/ukrainian/ukraine_in_russian

Given I'm on the url `ukrainian/ukraine_in_russian`
when I look at the navigation
Then is it is the same navigation as /ukrainian
and the `PYC` should be highlighted
```

```
Given I use a screen reader
When I activate the ‘skip to content’ link
Then the h1 in the main region should be announced in Russian
And after this announcement, I must be able to navigate to the next piece of content below the h1

Given I use a screen reader
When I navigate to the header
The content should be announced in Ukrainian (`uk`)

Given I use a screen reader
When I navigate to the main content
The content should be announced in Russian (`ru-UA`)

Given I use a screen reader
When I navigate to the footer
The content should be announced in Ukrainian (`uk`)
```

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

**Comment by DenisHdz** [comment]
❓ = Not done
✅ = Done, pass
❌ = Done, fail
👀 = Review with UX

## Checklist - Manual ##

Following this test guide: https://bbc.github.io/accessibility-news-and-you/accessibility-news-and-testers

  - ### Design ✅ 
 UX Approved

  - ### Color contrast ✅
 Checking the h1
![image](https://user-images.githubusercontent.com/30599794/86231713-ddd7a680-bb8a-11ea-87e5-551e32dbb839.png)

  - ### Color blindness ✅ 
 Manually looked through the different colour variations with `chromeLens` plugin:
![image](https://user-images.githubusercontent.com/30599794/86232189-8259e880-bb8b-11ea-979b-aa2e9fb5f5a2.png) 
And the pages' components are still visible

  - ### High Contrast mode PC ✅ 
![hcm](https://user-images.githubusercontent.com/46446236/86235923-1f6b5000-bb91-11ea-88a1-ea25a4dff2df.jpg)

  - ### Color preferences changed in Firefox ✅ 
Background, text, unvisited/visited links changed colour 
![image](https://user-images.githubusercontent.com/30599794/86233889-01e8b700-bb8e-11ea-954f-e1c0cb801f2e.png)

    
  - ### Text resized to 200% ✅ 

  - ### Page zoomed to 200% ✅ 

  - ### Heading order ✅ 

  - ### Icons ✅ 
   
  - ### Keyboard navigation: tab order, focus css, trace tab path ✅ 
![image](https://user-images.githubusercontent.com/30599794/86235838-fb0f7380-bb90-11ea-9ba9-78337ac8acb6.png)


  - ### aXe ✅ 
![image](https://user-images.githubusercontent.com/30599794/86236012-4164d280-bb91-11ea-8aa9-4d98cf2ffcea.png)
 issue covered here: dequelabs/axe-core#1597

  - ### No CSS ✅ 
 Same as front page 

  - ### No Javascript ✅ 
 Page still renders but no images after the first image is rendered
![image](https://user-images.githubusercontent.com/30599794/86235664-aec43380-bb90-11ea-9868-30ee69f291b6.png)
 

  - ### bbc-a11y ✅ 
![image](https://user-images.githubusercontent.com/30599794/86235391-41180780-bb90-11ea-92c8-a35a5cf4c87b.png)



## Checklist - Assistive Technology ##

 The h2 offscreen text for top stories is “Top story - Ukraine in Russian” - **This is in the data in CPS: ‘strapline.name’ of the first group for https://www.bbc.com/ukrainian/ukraine_in_russian.json. We need editorial to update it** -> `Главное` ✅

 - ###  VoiceOver OS (Mac) (Screen Reader) 

   > With Safari (Latest Version) on Mac OS (Latest Version)  

  and

 - ### VoiceOver iOS (iPhone/iPad) (Screen Reader) 

    > Latest Version with Safari (Latest Version) on iOS (Latest Version)

      - Ukrainian is not supported by VoiceOver https://support.apple.com/en-us/HT206175

    -  When setting the laptop language as both Ukrainian and Russian, the main content and visually hidden text are all pronounced correctly in Russian. However, navigation and footer are pronounced like: “Cyrillic capital letter P, Cyrillic capital letter O” because Ukrainian is not supported. This behavior is present for both Ukrainian and Russian as machine languages. Hence, the whole experience of using this IDX page with a screen reader is quite bad with Apple. - **CG:  I guess that makes sense if VoiceOver doesn’t support Ukrainian, we can however see that the language attributes are doing the job that they should be doing, which is telling the screen reader that the language has changed and what language this content is now in. If a user is using NVDA this should work as expected as both languages are supported https://www.nvaccess.org/files/nvda/documentation/userGuide.html#Internationalization**

     - Russian translation for `relatedContent` isn't included in the `russianMainTranslations` - **PR to fix it https://github.com/bbc/simorgh/pull/7095** ✅

     -  On iPhone, if the phone language is Russian, timestamps are pronounced in Russian two times correctly. However, when device language is Ukrainian, then first time, the timestamp is pronounced correctly in Russia, whereas the second time it has a weird combination on English and Ukrainian like “Twenty(in English) May(in Ukrainian) Twentytwenty(in English)”. - **CG: I guess some weirdness is to be expected if Ukrainian isn’t supported by voiceover currently**

 - ### TalkBack  (Screen Reader) 
 
   > Latest Version with Chrome (Latest Version) on Android (Latest Version)

     - Phrases like “Heading 1” are pronounced in English with Ukrainian accent + the number is pronounced in Ukrainian - **CG: Ok guess we can’t do anything about that**

     - In the footer screen reader said “BBC Ukraine - link not in the list” - **CG: Could say ‘not in list’ to indicate that you’ve just come out of a list as the previous content was list content?**

     - In the footer for different links, BBC is pronounced differently for some reason (don’t know if it’s a problem). E.g. “Зв’язатися з BBC” BBC is pronounced correctly as in English. But in “© 2020 BBC. ВВС не несе відповідальності за контент інших сайтів.” BBC is pronounced as Ukrainian letters.
     **We need to check with Editorial if they would expect `BBC` to be announced in English or in the site language** - Editorial: `In speech it would be BBC, pronounced as in English, irrespective of what script is used.` -> **Issue to fix it https://github.com/bbc/simorgh/issues/7112**

- ### Windows AT
  Can't install the Ukrainian language as it needs admin access

**Comment by DenisHdz** [comment]
Closing it as no P1s came out of the swarm.
